src/components/SliderButton.tsx

```diff
@@ -25,7 +25,7 @@ export default function WalletImageSlider({
   }
 
   return (
-    <div className="flex flex-col justify-center w-full z-40 absolute bottom-0">
+    <div className="flex-nowrap flex-col justify-center w-full float-left z-40 absolute bottom-0">
```

Comment:
This seems strange to me, you're using flex box and also using float here. From my understanding, flexbox can't use floats.

Reply:
leave behind from changes. Removed float-left.